test/auth.test.js

```diff
@@ -75,15 +77,16 @@ describe('Login', () => {
   });
 
   it('should not login an unexisting user', (done) => {
-    chai.request(server)
+    chai
+      .request(server)
       .post('/api/auth/login')
       .send({
         email: 'alain666326@gmail.com',
-        password: 'password',
+        password: 'password'
       })
       .end((err, res) => {
         res.should.have.status(404);
-        res.body.should.have.property('message', 'user doesn\'t exist');
+        res.body.should.have.property('message', "user doesn't exist");
```

Comment:
@alainburindi  have to modify this message I think. It doesn't make sense from a user point of view